app/views/settings/_default_assign.erb

```diff
@@ -40,14 +40,21 @@
    <%= check_box_tag('settings[interactive_assignment]', true, interactive_assignment) %>
  </p>
 
- <p><%= l(:self_assignment_explanation) %></p>
- <p>
-   <%= label('settings[self_assignment]', l(:self_assignment)) %>
-   <%-
-     self_assignment = @settings['self_assignment'] || 'false'
-     self_assignment = (self_assignment == 'true')
-   -%>
-   <%= hidden_field_tag('settings[self_assignment]', false) %>
-   <%= check_box_tag('settings[self_assignment]', true, self_assignment) %>
- </p>
+  <p><%= l(:self_assignment_explanation) %></p>
+  <p>
+    <%= label('settings[self_assignment]', l(:self_assignment)) %>
+    <%-
+      self_assignment = @settings['self_assignment'] || 'false'
+      self_assignment = (self_assignment == 'true')
+    -%>
+    <%= hidden_field_tag('settings[self_assignment]', false) %>
+    <%= check_box_tag('settings[self_assignment]', true, self_assignment) %>
+  </p>
```

Comment:
Oh, was there only 1 previously? If so, could you re-indent the whole file a separate commit, so that the code changes don't get lost in the whitespace change?